### ISSUE TITLE
Dancing keys

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#26561]
Does this issue occur when all extensions are disabled?: Yes

```
Version: 1.80.0-insider
Commit: 660393deaaa6d1996740ff4880f1bad43768c814
Date: 2023-07-04T10:58:02.000Z
Electron: 22.3.14
ElectronBuildId: 21893604
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Linux x64 6.2.6-76060206-genericSteps to Reproduce:
```
I am using a QWERTZ keyboard with German layout. When doing `Ctrl + < chording to -` to trigger a corresponding (chorded) shortcut, I am getting `Ctrl + <, /` is not a command. So the shortcut which I defined using `-` is interpreted as using `/`. Similarily, undoing with `Ctrl + z` doesn't work while `Ctrl + y` does, so `z` is now interpreted as `y`. It's quite confusing, and surprising because I was able to define these shortcuts using the exact same keys.

In a nutshell, keys hits which produce that correct character on screen in VSCode as not identified properly in shortcuts.
